### DISCUSSION TITLE
Customizing energy dashboard

### DISCUSSION THREAD

**Discussion by Rantaki** [home-assistant/frontend#15918] [Dashboard]
It would be great to have an opportunity to add new energy sources to the energy dashboard. Well to be exact, it might be enough to be able to rename the existing sources. Around the world there are plenty of different energy sources in use and the current dasboard doesn’t support them. For example, in Finland there is a very popular energy source used for heating: the energy is delivered as hot water from factory and measured in kwh. Currently I follow it on the energy dashboard as gas as there is no suitable source for it.

**Comment by botihu** [comment]
I would like to have add two more energy line values to the dashboard (both of then could be calculated on the fly, I don't think for storing them: Household locally consumed from solar yield (there is a % gauge, but I'd like to have a kWh value for the same) and the combined energy was used by the house [ie: how much energy you would have to buy from the grid if the grid would be the only energy source for your household energy needs])

**Comment by ChristophCaina** [comment]
at all, the energy dashboard could really get further improvements:
-> add autarcy information of solar installation
-> add self-sufficiency of solar installation
-> add oil usage for heating
-> add other heating options
-> split water usage into cold & warm water
